app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title =  t "layouts.application.title_app"
+    if page_title.empty?
+      base_title
+    else
+      [page_title, base_title].join(" | ")
+    end
```

Comment:
dùng toán tử 3 ngôi đoạn này nhé.
phân biệt thêm: nil?, empty?, blank?, present? giúp a

Reply:
nil dùng cho object
empty dùng cho array, hash, string
một object được coi là blank nếu nó nil, false, hay empty hoặc dùng cho 1 chuỗi chỉ là các khoảng trắng
present là hàm phủ định của blank

Reply:
vậy trong trường hợp này nên dùng cái nào e?

Reply:
e nghĩ là dùng empty ạ